express.install

```diff
@@ -196,3 +196,11 @@ function express_update_7005() {
   variable_set('express_permissions_owner_cap', 5);
 }
 
+/**
+ *
+ * Enable erus_lite module.
+ */
+function express_update_7006() {
+  // Enable module.
+  module_enable(array('update','erus_lite'));
```

Comment:
Same here. I like the idea of beta testing the ERUS functionality with Atlas commands instead of initially turning it on service-wide.